### PR TITLE
Add new modules to CS Connections landing page

### PR DESCRIPTION
**Pull request by kelbyhawn**
Updates to https://code.org/educate/csc:
- Add four new modules to the Modules section under the Poem Art box
- Add a "Feedback" section that links to a Google Form
- Change `h2`s: _Open-ended projects_ > _Sample Projects_ & _Bridging Lessons_ > _Quick Activities_
- Remove images and add "Sample" to each title in the Sample Projects section

**Relevant info:** [Asana task](https://app.asana.com/0/1202773141567416/1202934806545633) • [Gdoc w/ details](https://docs.google.com/document/d/1jZIkCcjIigAuQzBq92P26EJ_ImhFO3hMYBh6iyRktiY/edit)

----

### Before:
Screen Shot 2022-09-08 at 3 57 51 PM
Screen Shot 2022-09-08 at 3 57 38 PM

### After:
Screen Shot 2022-09-08 at 3 58 06 PM
Screen Shot 2022-09-08 at 3 57 12 PM

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/css/csc-module.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    font-family: 'Gotham 4r', sans-serif;
+    color: $color-black !important;
+    text-decoration: none !important;
+  }
+
+  .csc-module-wrapper {
+    padding: 0 1.5em 1.5em;
+  }
+
+  h3{
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/code.org/public/css/csc-module.scss]: small nit: should there be a space here (same for `a.btn` below)?

File: pegasus/sites.v3/code.org/public/css/csc-module.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  padding: 1em;
+
+  .csc-module-outer-wrapper {
+    border: 1px solid;
+    border-radius: 8px;
+  }
+
+  a {
+    font-family: 'Gotham 4r', sans-serif;
+    color: $color-black !important;
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/code.org/public/css/csc-module.scss]: why do we need important here and below? (we definitely might just curious why!)

File: pegasus/sites.v3/code.org/public/educate/csc.md.partial [hunk 3]
```diff
@@ -26,23 +39,35 @@ They're perfect for teachers who'd like to incorporate computer science into the
 
 {{ tutorial_promo2_single_course, heading: "", background_image: "/images/fit-1940/csc/poetry/tutorial_promo2_background_poetry.jpg", text_color: "white", grades: "3+", subject: "ELA", duration: "5 hours", box_background_color: "rgb(0,0,0,0.4)", right_heading: "Coding with Poetry", right_description: "Connect computer science and poetry in your classroom with this series of lessons.", right_specs: "", right_button_url: "https://studio.code.org/s/poetry-2021/?viewAs=Teacher", right_button_text: "View module" }}
 
-## Open-ended projects
+{{ csc_module, title: "Marine Life Ecosystems", image: "/images/csc/landing/module_marine_life.png", grades: "3-5", subject: "Science", duration: "4 hours", description: "Restore balance to a marine life ecosystem on the verge of overfishing.", button_text: "Learn more", url: "https://studio.code.org/s/csc-ecosystems-2022" }}
+{{ csc_module, title: "Star Quilts", grades: "3-5", image: "/images/csc/landing/module_star_quilts.png", subject: "Math", duration: "4 hours", description: "Design a Native American star quilt using rhombus patterns in The Artist.", button_text: "Learn more", url: "https://studio.code.org/s/csc-starquilts-2022" }}
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/code.org/public/educate/csc.md.partial]: another tiny nit (sorry!): can we keep the properties in the same order? so `grades` after `image` here to be consistent with the others? I thought `grades` wasn't set on this one at first

**Review by bethanyaconnor — APPROVED** [review]
A couple of tiny nits and a question but otherwise lgtm!

File: pegasus/sites.v3/code.org/public/css/csc-module.scss (hunk 1 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/css/csc-module.scss]: Ah yes! Good catch 😄

File: pegasus/sites.v3/code.org/public/css/csc-module.scss (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/css/csc-module.scss]: I think this was leftover from my earlier refactor process — just tried it without and we don't need them! ✨

File: pegasus/sites.v3/code.org/public/educate/csc.md.partial (hunk 3 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/educate/csc.md.partial]: No need to apologize, I love it! Just updated.

**Pull request merged**